src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -0,0 +1,92 @@
+import { XXXL } from "@zendeskgarden/react-typography";
+import type { ServiceCatalogItem } from "./data-types/ServiceCatalogItem";
+import styled from "styled-components";
+import { Button } from "@zendeskgarden/react-buttons";
+import { useState } from "react";
+import ChevronUp from "@zendeskgarden/svg-icons/src/16/chevron-up-fill.svg";
+import ChevronDown from "@zendeskgarden/svg-icons/src/16/chevron-down-fill.svg";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+import { useTranslation } from "react-i18next";
+
+const ItemTitle = styled(XXXL)`
+  font-weight: ${(props) => props.theme.fontWeights.semibold};
+`;
+
+const CollapsibleDescription = styled.div<{ expanded: boolean }>`
+  font-size: ${(props) => props.theme.fontSizes.md};
+  text-align: left;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: ${(props) => (props.expanded ? "none" : 3)};
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
+  margin-top: ${(props) => props.theme.space.md};
+`;
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 120px;
+
+  @media (max-width: 768px) {
+    flex-direction: column;
+  }
+`;
+
+const LeftColumn = styled.div`
+  flex: 2;
+  border-bottom: ${(props) => props.theme.borders.sm}
+    ${(props) => getColorV8("grey", 300, props.theme)};
+  padding-bottom: ${(props) => props.theme.space.lg};
```

Comment:
Are we sure that we always want to show a bottom border even if no options are available? I think the grey line is more like a separator between the description and the options, but we don't have a design for a page without options. 

In any case these border and padding should not be set on the whole column but only on the first part, but it is ok to do it in the PR where we will introduce the options

Reply:
Ok I will  work more on that in the PR with form fields